tests/test_coordinator.py

```diff
@@ -93,49 +93,93 @@ def coordinator(project, deployer, stake_info, flat_rate_fee_model, initiator):
     return contract
 
 
+@pytest.fixture()
+def global_allow_list(project, deployer, coordinator):
+    contract = project.GlobalAllowList.deploy(
```

Comment:
Should we add tests (maybe not in this file) for the actual `GlobalAllowList` contract?